pyproject.toml

```diff
@@ -36,6 +36,8 @@ httpx = "^0.27.0"
 retrying = "^1.3.3"
 rich = "^13.7.1"
 geojson = "^3.0.1"
+botocore = "^1.35.0"
```

Comment:
please also declare this as an optional dependency (` {version = "^1.36.11", extras = ["s3"]}`)

Reply:
Updated :) Both are now optional dependencies.